package/python-kiwi-spec-template

```diff
@@ -372,12 +387,6 @@ Requires:       python%{python3_pkgversion}-lxml
 Requires:       python%{python3_pkgversion}-requests
 Requires:       python%{python3_pkgversion}-setuptools
 Requires:       python%{python3_pkgversion}-xmltodict
-%if 0%{?rhel} || 0%{?fedora}
-Requires:       (python%{python3_pkgversion}-typing-extensions if python%{python3_pkgversion} < 3.8)
-%endif
-%if 0%{?suse_version}
-Requires:       (python%{python3_pkgversion}-typing_extensions if python%{python3_pkgversion} < 3.8)
-%endif
```

Comment:
The current code base runs perfectly fine also with py3.6. Can we keep this in the community spec file please until we have a real reason in code that breaks it. It's a personal wish because I'm using this spec file for maintaining the package in TW and ALP and it would be nice if I can continue to do this. As all other distro maintainers build their own specs around the code I think it really doesn't hurt if you keep it. Thanks much

Reply:
poetry-core is in Leap, poetry proper is not

Reply:
I've updated the spec to force the Python 3.11 stack for Leap 15.